### ISSUE TITLE
Version 0.1.4 contains breaking changes

### ISSUE THREAD

**Issue by marius-wieschollek** [mazira/base64-stream#10]
In version 0.1.3, a call like this worked:
```
const base64Stream = require('base64-stream');
const stream = base64Stream.decode();
```

Now with version 0.1.4 you need to write it like this:
```
const {Base64Decode} = require('base64-stream');
const stream = new Base64Decode();
```

The original way to create a stream does not work anymore which makes this a breaking change because other packages need to adapt their code in order to work with this version. 
But NPM just treats this as a bugfix release and just goes for the new version instead of the one specified in the package.json.

In my case i am using `pageres-cli` which depends on `screenshot-stream` which is now broken.

So i would like to ask you to unpublish the version and re-release it as a version 1.X or so to make sure 
NPM sees it as a new version with breaking changes and does not simply install it like it does now.

**Comment by rossj** [comment]
Oh gosh, sorry about that. It looks like 0.1.3 was published almost 3 years ago, and changes have occurred in the repository since then that were not published to NPM. I forgot about this and assumed things were in sync - that the recent change was only removing test files.

I've re-published 0.1.3 as 0.1.5, and published the latest as 1.0.0.

**Comment by marius-wieschollek** [comment]
That seems to have fixed it. Thank you!
